src/services/Twitter.js

```diff
@@ -17,6 +17,52 @@ const clientTwitter = new Twit({
   access_token_secret: TWITTER_ACCESS_TOKEN_SECRET,
 });
 
+/**
+* Post thread, each tweet can have text and photos or only text
+*
+* @async
+* @param {Array<Object>} tweetSeq
+*/
+const uploadThreadTwitter = (tweetSeq, tweetId = '') => {
+  if (tweetSeq.length === 0) {
+    return;
+  }
+
+  const tweetToSend = tweetSeq.shift();
+
+  const uploadedMedia = tweetToSend.media !== undefined
```

Comment:
Regarding `tweetSend.media` issue, `uploadThreadTwitter` should not return when media is not present, because tweets can have media attached to it or not. 
The other suggestion was taken into account and it's commited (https://github.com/vostpt/bot/pull/134/commits/45fb981742f6b3b6a3425e43b51d65a2eb2eba36).

Thanks for the feedback!